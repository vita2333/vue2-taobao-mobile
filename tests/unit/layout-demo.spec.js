import Demo from '../demo/layout'
import { snapshotDemo } from '../index'

snapshotDemo(Demo)
