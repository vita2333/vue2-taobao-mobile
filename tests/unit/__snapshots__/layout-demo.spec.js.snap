// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div class="vita-row">
    <div class="vita-col vita-col--2">span 2</div>
    <div class="vita-col vita-col--4">span 4</div>
  </div>
  <div class="vita-row" style="margin-left: -6px; margin-right: -6px;">
    <div class="vita-col" style="padding-left: 6px; padding-right: 6px;">gutter 12</div>
    <div class="vita-col" style="padding-left: 6px; padding-right: 6px;">gutter 12</div>
  </div> <span class="vita-row vita-row--flex vita-row--align-left vita-row--justify-space-between"><div class="vita-col">flex left space-between</div> <div class="vita-col">flex left space-between</div></span>
  <p class="vita-row vita-row--flex vita-row--align-bottom">
    <div class="vita-col">flex bottom</div>
    <div class="vita-col">flex bottom</div>
  </p>
</div>
`;
